resources/views/footer-copyright.blade.php

```diff
@@ -2,11 +2,15 @@
     'isArkProduct'  => true,
     'copyText'      => 'ARK.io | ' . trans('ui::general.all_rights_reserved'),
     'copyrightSlot' => null,
+    'privacyPolicyUrl' => null,
+    'cookiePolicyUrl' => null,
 ])
 
 <div {{ $attributes->merge(['class' => 'flex flex-col py-6 space-y-2 font-semibold text-sm text-theme-secondary-700 sm:flex-row sm:space-y-0 sm:space-x-1'])}}>
     <span class="whitespace-nowrap">
         {{ date('Y') }} &copy; {{ $copyText }}
+        @if ($privacyPolicyUrl) <br class="sm:hidden"> <span class="hidden sm:inline">| </span><a class="underline" href="{{ $privacyPolicyUrl }}" target="_blank">@lang('ui::general.privacy_policy')</a> @endif
```

Comment:
might be better to move this after the "isArkProduct" and "copyrightSlot" values, so it won't look odd when it's used on another project other than MSQ